crowdin.yml

```diff
@@ -22,9 +22,7 @@ files: [
   #
   "languages_mapping" : {
     "two_letters_code" : {
-      "es-ES" : "es",
```

Comment:
Is it intentional that these are removed?

Reply:
Yes, this is the second commit. I have observed that they are not needed. The two letters code of es-ES is es by default. This file is only a hack to get the locale code on languages that we need them when using the two letters code.
Maybe there is something I'm missing?

Reply:
Ok, I didn't realise that you meant languages separate from the pt / pt-BR change. All good then.